### PR TITLE
adding interlude audio dt overlay file

### PR DESCRIPTION
**Pull request by Yash-BlueRockSoft**

### CODE REVIEW

**Comment by benpayne** [comment]
You have some build checks that are failing.  the overlay one is about documation that I took care of for the digital hat.  But looks like we have a build failure also, we should fix this so that we are ready to submit upstream.

File: arch/arm/boot/dts/overlays/Makefile
Comment (benpayne) [inline arch/arm/boot/dts/overlays/Makefile:1]: the added file should be in the middle, alphabetical order.  Also I think you need a space between the last char and the back slash, that can cause issues.  Also don't need to add new blank lines...

File: arch/arm/boot/dts/overlays/interludeaudio-analog-overlay.dts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+			i2s-controller = <&i2s>;
+			status = "okay";
+
+			simple-audio-card,name = "Interlude Audio Analog";
+
+			simple-audio-card,widgets =
+				"Line", "Line In",
+				"Line", "Line Out";
+
+			simple-audio-card,routing =
```
Comment (benpayne) [inline arch/arm/boot/dts/overlays/interludeaudio-analog-overlay.dts:21]: Do we really want "Line out", to have 4 values?  We don't support differential.  What benefit does this provide the end user?

Comment (Yash-BlueRockSoft) [reply arch/arm/boot/dts/overlays/interludeaudio-analog-overlay.dts:21]: those are audio routing paths for the DAPM dont believe the user gets to select those or make changes to them

**Comment by benpayne** [comment]
You should remove the changes to the GitHub action, we don't want to change that file.  Also you have a lot of good clean up in the 8804.c file, but you shouldn't have work like that on a merge request for other work.  So I'd put in a new feature in JIRA to clean up the 8804.c file and then create a branch for that work and a separate merge request.  If you remove both of these changes I'll approve the merge request.

**Comment by Yash-BlueRockSoft** [comment]
> 

yeah made the changes to the github workflows also for the wm8804 i think we should let it complete with the current changes as reverting back those changes and putting them in a new branch would essentially mean reworking those things on a different branch moving forward we can create jira features for changes and merge them using a different branch?

**Pull request closed without merging**